pkg/client/cli/intercept/state.go

```diff
@@ -329,34 +329,43 @@ func (s *state) CreateRequest(ctx context.Context) (*connector.CreateInterceptRe
 	return ir, nil
 }
 
+func getArg(args []string, flag string) (string, error) {
+	feq := flag + "="
+	for i, arg := range args {
+		var v string
+		switch {
+		case arg == flag:
+			i++
+			if i < len(args) {
+				if v = args[i]; strings.HasPrefix(v, "-") {
+					v = ""
+				}
+			}
+		case strings.HasPrefix(arg, feq):
+			v = arg[len(feq):]
+		default:
+			continue
+		}
+		if v == "" {
+			return "", errors.New("docker flag `--name` requires a value")
+		}
```

Comment:
This shouldn't be `--name`, but should rather include the provided `flag`

Reply:
Haha, yeah :-)